src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -1,5 +1,6 @@
 (ns puppetlabs.puppetserver.certificate-authority
-  (:import [org.apache.commons.io IOUtils]
+  (:import (java.util.concurrent.locks ReentrantReadWriteLock)
```

Comment:
The style of `[]` vs `()` is inconsistent here.  Cursive automatically uses `()`.  I can make this consistent.  Thoughts?

Reply:
`()` is more idiomatic, and I'd be into making them all lists. IIRC, this code was written prior to strong standards in the clj community.